### PR TITLE
Add crystalia site and ontology

### PR DESCRIPTION
**Pull request by vladistan**

### CODE REVIEW

**Comment by davidlehn** [comment]
Can you please add your github id for yourself and any other maintainers to the README?

File: crystalia/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Crystalia ontology
+
+This is the permalink for Crystalia Ontology [https://w3id.org/crystalia](https://w3id.org/crystalia)
+The original link is [https://crystalia.link](https://crystalia.link)
+
+# Contact
+
+Vlad Korolev (vkorol1@umbc.edu)
+University of Maryland, Baltimore County
```
Comment (TallTed) [inline crystalia/README.md:9]:
```suggestion
Vlad Korolev (vkorol1@umbc.edu) <br>
University of Maryland, Baltimore County <br>
```

**Comment by vladistan** [comment]
> Can you please add your github id for yourself and any other maintainers to the README?

Done.  Also squashed all the commits

File: crystalia/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Crystalia ontology
+
+This is the permalink for Crystalia Ontology [https://w3id.org/crystalia](https://w3id.org/crystalia)
+The original link is [https://crystalia.link](https://crystalia.link)
+
+# Contact
+
+Vlad Korolev (vkorol1@umbc.edu) <br>
+Github ID:  vladistan
```
Comment (davidlehn) [inline crystalia/README.md:9]: Match the other style:
```suggestion
Github ID:  vladistan <br>
```

**Pull request merged**